operations/finance.md

```diff
@@ -347,80 +404,52 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
-| `AccountingItems` | array of [Accounting item](finance.md#accounting-item) | optional | The accounting items. |
-| `CreditCardTransactions` | array of [Credit card transaction](finance.md#credit-card-transaction) | optional | The credit card payment transactions. |
+| `OrderItems` | array of [Order item](#order-item) | optional | The accounting items. |
+| `PaymentItems` | array of [Payment item](#payment-item) | optional | The accounting items. |
+| `CreditCardTransactions` | array of [Credit card transaction](#credit-card-transaction) | optional | The credit card payment transactions. |
 
-#### Accounting item
+#### Order item
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
 | `Id` | string | required | Unique identifier of the item. |
-| `CustomerId` | string | required | Unique identifier of the [Customer](customers.md#customer) whose account the item belongs to. |
+| `AccountId` | string | required | Unique identifier of the account (for example [Customer](customers.md#customer)) the item belongs to. |
 | `OrderId` | string | optional | Unique identifier of the order \(or [Reservation](reservations.md#reservation) which is a special type of order\) the item belongs to. |
```

Comment:
It says optional by in connector DTO OrderId is not nullable as the rest of the optionals, do we want to consolidate it?